### PR TITLE
Implement Api

### PR DESCRIPTION
**Pull request by lmachens**
This pull request adds an API to access matches.

Endpoints:
GET `/api/matches` returns all matches
POST `/api/matches` adds a new match
POST `/api/matches/{id}/highlights` adds a highlight to a match by Id

**Breaking**
Add `PORT` environment variable.
Add `MONGODB_URI` environment variable.

### CODE REVIEW

**Review by webdevelukas — APPROVED** [review]
lgtm 👍

File: samples.http [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@baseUrl = http://localhost:3001/api
```
Comment (AlexisRoe) [inline samples.http]: problematic to hardcode the port of the local server here, better to use the environment variable directly:
{{$processEnv [%]envVarName}} --> https://github.com/Huachao/vscode-restclient
```suggestion
@baseUrl = http://localhost:{{$processEnv [%]PORT}}/api
```

**Review by AlexisRoe — CHANGES_REQUESTED** [review]
environment variables should be considered to be more flexible and congurant (?)

File: src/server.ts [hunk 2]
```diff
@@ -2,9 +2,14 @@ import dotenv from 'dotenv';
 dotenv.config();
 
 import express from 'express';
+import { connectToMongoDb } from './lib/db';
 import router from './lib/router';
 
-const { PORT = 3001 } = process.env;
+const { PORT = 3001, MONGODB_URI } = process.env;
```
Comment (AlexisRoe) [inline src/server.ts]: could we get rid of it, in favour of direct imported env variables, after changing samples.http?

**Review by AlexisRoe — COMMENTED** [review]
;-)

**Review by AlexisRoe — APPROVED** [review]
gj, thanks

**Pull request merged**